src/frequenz/client/microgrid/_component_data.py

```diff
@@ -87,6 +87,24 @@ class MeterData(ComponentData):
     * `-` means supply into the grid.
     """
 
+    reactive_power: float
+    """The 3-phase reactive power, in Volt-Ampere Reactive (VAr).
```

Comment:
Maybe it's better to be explicit here?
```suggestion
    """The total 3-phase reactive power, in Volt-Ampere Reactive (VAr).
```